### PR TITLE
Refactor VR Harp addresses

### LINKED ISSUE #95: Clashes in register addresses between outdoor and VR data schemas
According to the README, the address 211 is reserved to the Pupil Labs IMU stream:

https://github.com/emotional-cities/pluma-experiments/blob/43b246217ce1f096f1c8af4a279e91560a345707/README.md?plain=1#L188

Even though this stream is currently not in use, it might be a good idea to avoid the clash with the VR georeference stream timestamps, currently also targeting 211. Moving this stream to address space ~220 would also make it more clear the relationship with the other unity streams. It looks like 221 is currently free, but requires confirmation.

Comparing directly the [outdoor](https://github.com/emotional-cities/pluma-analysis/blob/main/pluma/schema/outdoor.py) and [VR](https://github.com/emotional-cities/pluma-analysis/blob/main/pluma/schema/vr.py) schemas, we can find other address clashes, specifically 227 and 228 containing protocol-level registers. For these it might be wise to reserve a range of addresses dedicated to experiment-specific registers (as opposed to hardware device registers). These could live in the address range [180-199] which should not be currently used by anything.

### PR DESCRIPTION
**Pull request by aspaNeuro**
This PR change the address space for VR sensors:
- VRTransform from 219 to 180
- GeoReference 211 to 181
- ScreenCapture 220 to 182
- VR Experiment protocol [225-228] to [185-188]
- Glia [215-218] to [190-193]
- Update Readme

Fix #95

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,12 +191,13 @@ To achieve this, Bonsai is randomly toggling a digital output in the HARP behavi
...
-|                           |         Mouth         |   218    |                |            Timestamped(long[])                |
-|     **VRTransform**       |       VrTimestamp     |   219    |                |            Timestamped(long)                  |
-|     **UnityImage**        |       VrTimestamp     |   220    |                |            Timestamped(long)                  |
+|       **Omnicept**        |      EyeTracking      |   190    |                |            Timestamped(long[])                |
+|                           |       HeartRate       |   191    |                |            Timestamped(long[])                |
+|                           |          IMU          |   192    |                |            Timestamped(long[])                |
+|                           |         Mouth         |   193    |                |            Timestamped(long[])                |
+|     **VRTransform**       |       VrTimestamp     |   180    |                |            Timestamped(long)                  |
+|    **GeoReference**       |       VrTimestamp     |   181    |                |            Timestamped(long)                  |
```
Comment (glopesdev) [inline README.md]:
```suggestion
|    **Georeference**       |       VrTimestamp     |   181    |                |            Timestamped(long)                  |
```

Check if all other instances of `Georeference` throughout are all lower-case for consistency.

**Comment by glopesdev** [comment]
@aspaNeuro one last comment regarding the protocol addresses:

> VR Experiment protocol [225-228] to [185-188]

These seem to have been renamed but are not included in the main README file. This could potentially make sense since they are protocol specific addresses, but should we then include them in an experiment-specific README somewhere in the repo?

File: README.md (hunk 1 above)
Comment (aspaNeuro) [reply README.md]: Fix in 66ac9552c5bc16c393219f1ebb1c57f132cea665

**Comment by aspaNeuro** [comment]
> @aspaNeuro one last comment regarding the protocol addresses:
> 
> > VR Experiment protocol [225-228] to [185-188]
> 
> These seem to have been renamed but are not included in the main README file. This could potentially make sense since they are protocol specific addresses, but should we then include them in an experiment-specific README somewhere in the repo?

They are in the main README I think. 

Are you saying to copy these as well to the https://github.com/emotional-cities/pluma-experiments/blob/main/VR-Alfama/README.md, I dont think this is relevant information for you to setup the VR.

The experimental protocol itself has some LSL messages that need documentation, I'm currently waiting for feedback from João Amaro on this and created an issue #96 for dealing with these.

**Review by jfrazao — APPROVED** [review]

**Comment by aspaNeuro** [comment]
> > @aspaNeuro one last comment regarding the protocol addresses:
> > > VR Experiment protocol [225-228] to [185-188]
> > 
> > 
> > These seem to have been renamed but are not included in the main README file. This could potentially make sense since they are protocol specific addresses, but should we then include them in an experiment-specific README somewhere in the repo?
> 
> They are in the main README I think.
> 
> Are you saying to copy these as well to the https://github.com/emotional-cities/pluma-experiments/blob/main/VR-Alfama/README.md, I dont think this is relevant information for you to setup the VR.
> 
> The experimental protocol itself has some LSL messages that need documentation, I'm currently waiting for feedback from João Amaro on this and created an issue #96 for dealing with these.

The commit cd9bfc8417976f31b312394495b1d11d81f2e09c fix this problem

**Review by glopesdev — APPROVED** [review]

**Pull request merged**